enginetest/queries/generated_columns.go

```diff
@@ -271,4 +415,42 @@ var GeneratedColumnTests = []ScriptTest{
 			},
 		},
 	},
+	{
+		Name: "adding a virtual column",
+		SetUpScript: []string{
+			"create table t1 (a int primary key, b int)",
+		},
+		Assertions: []ScriptTestAssertion{
+			{
+				Query:    "insert into t1 (a, b) values (1, 2), (3, 4)",
+				Expected: []sql.Row{{types.NewOkResult(2)}},
+			},
+			{
+				Query:    "alter table t1 add column c int generated always as (a + b) virtual",
+				Expected: []sql.Row{{types.NewOkResult(0)}},
+			},
+			{
+				Query:    "select * from t1 order by a",
+				Expected: []sql.Row{{1, 2, 3}, {3, 4, 7}},
+			},
+		},
+	},
+}
+
+var BrokenGeneratedColumnTests = []ScriptTest{
+	{
+		Name: "update a virtual column with a trigger",
+		SetUpScript: []string{
+			"create table t1 (a int primary key, b int, c int generated always as (a + b) virtual)",
+			"create table t2 (a int primary key)",
+			"create trigger t1insert before update on t1 for each row set new.c = 2",
+		},
+		Assertions: []ScriptTestAssertion{
+			{
+				// Not sure if this should be an error at trigger creation time or execution time
```

Comment:
at least need execution, could always alter a table column to/from a generated default

Reply:
did the set expression bounds check not fix this?

Reply:
It's a problem during plan building, that's when we verify no generated columns in DML